packages/cojson/src/SyncStateManager.ts

```diff
@@ -134,8 +152,3 @@ function getIsUploadCompleted(
 
   return true;
 }
-
-function simpleMemoize<T>(fn: () => T): () => T {
```

Comment:
Had to remove the memoization because was creating some issues in the subscribe tests